ci/builders/mac_ios_engine.json

```diff
@@ -89,12 +133,72 @@
                 "script": "flutter/sky/tools/create_full_ios_framework.py",
                 "language": "python3"
             },
+	    {
+                "name": "Profile-FlutterMacOS.framework",
```

Comment:
I think I asked this in another PR, but why is building the iOS artifacts called `FlutterMacOS.framework`?  FlutterMacOS.framework is the macOS embedder artifact...

Reply:
That was probably an attempt to use a better name than `artifacts`, which name should we use here? this is just what is shown in the step name.

Reply:
`Profile-Flutter.xcframework` or `Profile-ios-Flutter.xcframework` would be more accurate.

Reply:
Updated!